### ISSUE TITLE
Android users unable to reliably select alt text

### ISSUE THREAD

**Issue by pfrazee** [bluesky-social/social-app#2592]
Word is that some Android users can't get text selection to work. At times it seems that there's a secret spot in the UI that they have to aim for. Not clear why this is -- it worked for me in testing.

**Comment by haileyok** [comment]
guessing this had to do with the fact the selection and on press for expand kinda interfere. afaik should be resolved in that selection pr, although i admit to not testing much on android there. trick though was to move the on press to the text itself rather than wrap it as a pressable.

**Comment by Juicysteak117** [comment]
For whatever it's worth, I have never once been able to select alt text on Android to the point where I thought it simply wasn't implemented. I'd expect a long press to do it but it never has. Glad to know it's being looked into at least.
